### PR TITLE
NotImplementedError

### PR DESCRIPTION
**Pull request by Zernoxi**

### CODE REVIEW

File: content/showcase/_index.html [hunk 1]
```diff
@@ -197,6 +197,14 @@ <h1 class="title is-2">UQCS Showcase</h1>
                         link="https://github.com/sktapsall/CheckerBOT"
                         class="showcase-element has-text-centered"
                     >}}
+                    {{< showcase-item
+                        src="/img/media/showcase/nie.png"
```
Comment (nicklambourne) [inline content/showcase/_index.html:201]: You haven't added this image as part of your PR, when you do make sure it's exactly 1000px x 750px.

File: content/showcase/_index.html [hunk 2]
```diff
@@ -197,6 +197,14 @@ <h1 class="title is-2">UQCS Showcase</h1>
...
+                        title="UQCS2021-Team-NIE"
```
Comment (nicklambourne) [inline content/showcase/_index.html:202]: Title should be the name of the project, not the name of the team. See the existing entries at https://uqcs.org/showcase/

File: content/showcase/_index.html [hunk 3]
```diff
@@ -197,6 +197,14 @@ <h1 class="title is-2">UQCS Showcase</h1>
...
+                        caption="Hackathon the Experience. Wanted to make a game based on caring for hackers needs."
```
Comment (nicklambourne) [inline content/showcase/_index.html:203]: This could probably use a little more detail.

**Review by nicklambourne — CHANGES_REQUESTED** [review]
A few things to fix in this PR, but I loved the concept and the presentation.

File: content/showcase/_index.html (hunk 2 above)
Comment (Zernoxi) [reply content/showcase/_index.html]: Changed title to 'Hackathon the Experience'.

File: content/showcase/_index.html (hunk 3 above)
Comment (Zernoxi) [reply content/showcase/_index.html]: Added additional detail.

**Comment by nicklambourne** [comment]
Oh, and make sure your entry is at the top, we want to show off the more recent ones first! ❤️

File: content/showcase/_index.html (hunk 1 above)
Comment (Zernoxi) [reply content/showcase/_index.html:201]: How do I add the image to /img/media/showcase

**Review by Zernoxi — COMMENTED** [review]
Moved it to the top of the list.

**Comment by nicklambourne** [comment]
Awesome, looks like the last thing it needs is that cover image to be added at `/img/media/showcase/nie.png`

File: content/showcase/_index.html (hunk 1 above)
Comment (Zernoxi) [reply content/showcase/_index.html]: Added image.

File: content/showcase/_index.html [hunk 4]
```diff
@@ -12,6 +12,14 @@ <h1 class="title is-2">UQCS Showcase</h1>
         <div class="columns is-centered">
             <div class="column is-8">
                 <div class="showcase-gallery columns is-multiline">
+                    {{< showcase-item
+                        src="/img/media/showcase/nie.png"
```
Comment (nicklambourne) [inline content/showcase/_index.html:16]: One last thing (sorry!) it looks like your image is a jpg but this references a png. Either should be fine, it just needs to be consistent.

**Review by nicklambourne — CHANGES_REQUESTED** [review]

File: content/showcase/_index.html (hunk 4 above)
Comment (Zernoxi) [reply content/showcase/_index.html:16]:
```suggestion
                        src="/img/media/showcase/nie.jng"
```

Comment (Zernoxi) [reply content/showcase/_index.html]: Changed the extension.

**Review by nicklambourne — APPROVED** [review]
LGTM

**Review by TomCranitch — APPROVED** [review]
LGTM

**Pull request merged**